### PR TITLE
'harrasing' -> 'harassing'

### PR DESCRIPTION
**Pull request by ToxiClay**
#### Summary

SUMMARY: None

#### Purpose of change

This change addresses a misspelling I discovered throughout CDDA's codebase.

#### Describe the solution

I used Notepad++ to find and replace across the entire repository.

#### Describe alternatives you've considered

No alternatives.

#### Testing

Linting performed, though it seemed unnecessary.

#### Additional context

harrasing

### CODE REVIEW

File: lang/po/ar.po [hunk 1]
```diff
@@ -155548,7 +155548,7 @@ msgstr[5] ""
 #: lang/json/gun_from_json.py
 msgid ""
 "Emphasizing portability and agility over power, this all wood bow is "
-"suitable for small game and harrasing enemies."
```
Comment (Qrox) [inline lang/po/ar.po:155551]: `lang/po/xx.po` files are not supposed to be edited by hand.

Comment (ToxiClay) [reply lang/po/ar.po:155551]: Oh. Welp. Let me kill these and go again. Sorry

**Comment by ToxiClay** [comment]
Killing request due to manual edit of restricted files.

**Comment by ToxiClay** [comment]
Closing request due to manual edit of restricted files.

**Pull request closed without merging**